crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -726,6 +729,8 @@ where
         for c in &strategy.contracts {
             let op_name = get_contract_operation_name(provider, c, strategy.world_address).await;
             ui.print_sub(format!("{op_name} (class hash: {:#x})", c.diff.local_class_hash));
+            let contract_address = get_contract_address(c.salt, c.diff.base_class_hash, &[], c.contract_address);
+            ui.print_sub(format!("{op_name} (contract address: {:#x})", contract_address));
```

Comment:
Could we just re-work the output, to have like the contract name then the class hash and the contract address?

Currently we have something like this:
```
# Models (4)
  > register dojo_examples::actions::actions::moved (class hash: 0x91e05e8691a82229a17cf2b9b01b6f339fe704550151786b65734f0a9e45f1)
  > register dojo_examples::models::emote_message (class hash: 0x1080134c112da75344e02e4a6a9629778f5b7c0bf3f81055f1304bf75acaf75)
  > register dojo_examples::models::moves (class hash: 0x2e9c87df132573ed64a84b5d18059ba86cabef0ae0a4ccd211ce90ecb3dfce3)
  > register dojo_examples::models::position (class hash: 0x151d4e1655053bc12c41f2e46ff6cb740c024320f129d92331cb1d2e59e4aa3)
```
We may want something like:
```
# Models (4)
  > dojo_examples::actions::actions::moved
      class hash: 0x91e05e8691a82229a17cf2b9b01b6f339fe704550151786b65734f0a9e45f1
      contract address: 0x1080134c112da75344e02e4a6a9629778f5b7c0bf3f81055f1304bf75acaf75

...
```
WDYT?